### PR TITLE
optimze watchtower Dockerfile

### PR DESCRIPTION
**Pull request by Zoey2936**

### CODE REVIEW

File: Containers/watchtower/Dockerfile [hunk 1]
```diff
@@ -4,11 +4,9 @@ FROM containrrr/watchtower:1.5.3 as watchtower
 FROM alpine:3.17.3
 
 RUN apk add --no-cache bash
-COPY --from=watchtower /watchtower /
+COPY --from=watchtower /watchtower /watchtower
 
-COPY start.sh /
-RUN chmod +x /start.sh
+COPY --chmod=775 start.sh / start.sh
```
Comment (szaimen) [inline Containers/watchtower/Dockerfile:9]:
```suggestion
COPY --chmod=775 start.sh /
```

Comment (Zoey2936) [reply Containers/watchtower/Dockerfile:9]:
```suggestion
COPY --chmod=775 start.sh /start.sh
```
why not directly naming the end filename?

File: Containers/watchtower/Dockerfile [hunk 2]
```diff
@@ -4,11 +4,9 @@ FROM containrrr/watchtower:1.5.3 as watchtower
 FROM alpine:3.17.3
 
 RUN apk add --no-cache bash
-COPY --from=watchtower /watchtower /
+COPY --from=watchtower /watchtower /watchtower
```
[the remark this answers is not in the document]
Comment (Zoey2936) [reply Containers/watchtower/Dockerfile:7]: same as above

**Review by szaimen — APPROVED** [review]
LGTM

**Pull request merged**